IFC4x3/Sections/Resource definition data schemas/Schemas/IfcRepresentationResource/Entities/IfcTranslationOperation/DocEntity.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocEntity xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="IfcTranslationOperation" UniqueId="946a945b-8a4a-4e2c-9840-b56032727b5f" BaseDefinition="IfcCoordinateOperation" EntityFlags="32">
+	<Attributes>
+		<DocAttribute Name="Eastings" UniqueId="f2534a2f-e119-4a52-b1cf-a55c170eea45" DefinedType="IfcLengthMeasure" />
+		<DocAttribute Name="Northings" UniqueId="366b1fda-0ecf-4891-abb5-212d322b50fb" DefinedType="IfcLengthMeasure" />
+		<DocAttribute Name="OrthgonalHeight" UniqueId="caad8793-db34-4377-b751-b1591d8f3fe3" DefinedType="IfcLengthMeasure" AttributeFlags="1" />
```

Comment:
The naming is debatable, the types are weird.

- Easting/Northings *implies* UTM projection in some cases. Definitely not the best solution.
- Additionally, I thought that this entity could be used with `IfcGeographicCRS` - `IfcLengthMeasure` is false then, since the CRS uses angles.
- Typo `Orthogonal` 
- although I'd remove the prefix `Orthogonal` altogether and only use *Height* - it depends on the underlying `IfcCoordinateReferenceSystem` what this height is.
- What does `AttributeFlags` mean at height?

Proposal: 
- use `Axis1`, `Axis2` and `Axis3` as names for versality reasons. Or something similar.
- for type: use `IfcMeasureValue` with a WHERE rule checking for correct typing.